lib/cucumber/cli/options.rb

```diff
@@ -93,6 +94,8 @@ def parse!(args) # rubocop:disable Metrics/AbcSize, Metrics/MethodLength
 
         @args.options do |opts| # rubocop:disable Metrics/BlockLength
           opts.banner = banner
+          opts.on('--publish', 'Publish a report to https://reports.cucumber.io') { @options[:formats] << ['message', {}, CUCUMBER_PLUGIN_PUBLISH_URL] }
```

Comment:
Add authentication header here if `CUCUMBER_PUBLISH_TOKEN` is present. Like this:

https://github.com/cucumber/cucumber-jvm/blob/1ba0b255bdf3d4206146d6834096f69a446ffc4c/core/src/main/java/io/cucumber/core/plugin/PublishFormatter.java#L56

Reply:
done